.github/workflows/build.yml

```diff
@@ -193,14 +199,14 @@ jobs:
         env:
           PR: ${{ github.event.pull_request.number }}
       - name: Build
-        run: yarn ember build --environment production
+        run: pnpm ember build --environment production
       - name: Pack
         run: |
-          VERSION="$(jq -r '.version | split("+") | .[0]' package.json)"
+          VERSION="$(jq -r '.version' package.json)"
 
-          yarn pack
+          pnpm pack
           mkdir -p dist/npm
-          tar xvzf "ember-inspector-v$VERSION.tgz" -C dist/npm --strip-components 1
```

Comment:
We probably want to keep this name the same, right?

Reply:
it doesnt matter as its extracted anyway into dist/npm

Reply:
I misunderstood.
Pnpm pack generates a different filename

Reply:
Okay, just makes me worried changing names